generators/app/index.js

```diff
@@ -0,0 +1,265 @@
+        projectClassName: this.props.projectClassName,
+        projectCssClassName: this.props.cssClassName
+      }
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('tests/**/*'),
+      this.destinationPath('tests/')
+    );
+
+    this.fs.copy(
+      this.templatePath('_.babelrc'),
+      this.destinationPath('.babelrc')
+    );
+
+    this.fs.copy(
+      this.templatePath('_.eslintignore'),
+      this.destinationPath('.eslintignore')
+    );
+
+    this.fs.copy(
+      this.templatePath('_.eslintrc'),
+      this.destinationPath('.eslintrc')
+    );
+
+    this.fs.copy(
+      this.templatePath('_.github'),
+      this.destinationPath('.github')
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('_.gitignore'),
+      this.destinationPath('.gitignore'),
+      {
+        projectName: this.props.projectName
+      }
+    );
+
+    this.fs.copy(
+      this.templatePath('_.nvmrc'),
+      this.destinationPath('.nvmrc')
+    );
+
+    this.fs.copy(
+      this.templatePath('_.stylelintrc'),
+      this.destinationPath('.stylelintrc')
+    );
+
+    this.fs.copy(
+      this.templatePath('_.travis.yml'),
+      this.destinationPath('.travis.yml')
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('README.md'),
+      this.destinationPath('README.md'),
+      {
+        projectName: this.props.projectName,
+        titlecaseProjectName: this.props.titlecaseProjectName,
+        currentYear: new Date().getFullYear()
+      }
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('package.json'),
+      this.destinationPath('package.json'),
+      {
+        projectName: this.props.projectName,
+        projectClassName: this.props.projectClassName
+      }
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('CONTRIBUTING.md'),
+      this.destinationPath('CONTRIBUTING.md'),
+      {
+        projectName: this.props.projectName
+      }
+    );
+
+    this.fs.copy(
+      this.templatePath('CONTRIBUTORS.md'),
+      this.destinationPath('CONTRIBUTORS.md')
+    );
+
+    this.fs.copy(
+      this.templatePath('LICENSE'),
+      this.destinationPath('LICENSE')
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('NOTICE'),
+      this.destinationPath('NOTICE'),
+      {
+        currentYear: new Date().getFullYear()
+      }
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('RELEASE.md'),
+      this.destinationPath('RELEASE.md'),
+      {
+        projectName: this.props.projectName
+      }
+    );
+
+    this.fs.write(this.destinationPath('src/_mixins.scss'), '');
+    this.fs.write(this.destinationPath('src/_variables.scss'), '');
+
+    this.fs.write(this.destinationPath('docs/' + this.props.projectName + '.md'), '# ' + this.props.titlecaseProjectName + '\n\n' +
+      ' {insert description}\n\n' +
+      '## Getting Started\n\n' +
+      '- Install with [npm](https://www.npmjs.com): `npm install ' + this.props.projectName + '`\n' +
```

Comment:
We update this to reference yarn instead of npm for installing the terra component.
```
'- Install with [yarn](https://yarnpkg.com/): `yarn install ' + this.props.projectName + '`\n' +
```